hwdb.d/70-mouse.hwdb

```diff
@@ -143,6 +143,55 @@ mouse:*:name:*trackball*:*
 mouse:*:name:*TrackBall*:*
  ID_INPUT_TRACKBALL=1
 
+##########################################
+# 3D Connexion
+##########################################
+# From https://github.com/FreeSpacenav/spacenavd/blob/39856625a6de1e8c4b57c5938e1bf29d13cf1a9f/src/dev.c#L63
+mouse:usb:v256fpc603:name:*:*
```

Comment:
The list has too many entries than the referenced table... E.g. v256fpc603 is not listed in dev.c .

Reply:
You're right, I got confused by similar lists in PrusaSlicer (and derivatives) that have products ID be matched for both Logitech and 3D connexion. See the files touched by https://github.com/prusa3d/PrusaSlicer/pull/13688